src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,5 +1,34 @@
+  min468: '@media (min-width: 468px)',
+  min641: '@media (min-width: 641px)',
+  min969: '@media (min-width: 969px)',
+  min1131: '@media (min-width: 1131px)',
+};
+
+
+const pointBase = (width: string, height: string) => ({
+  width,
+  height,
+  [media.md]: { width: '0.45rem' },
+});
```

ESLint

[error] Expected pointBase to have a type annotation.

(@typescript-eslint/typedef)